lib/evaluation/limitDisclosureEvaluationHandler.ts

```diff
@@ -0,0 +1,158 @@
+import { Constraints, Descriptor, Field, Optionality, PresentationDefinition } from '@sphereon/pe-models';
+
+import { Status } from '../ConstraintUtils';
+import { JsonPathUtils } from '../utils/jsonPathUtils';
+
+import { AbstractEvaluationHandler } from './abstractEvaluationHandler';
+import { EvaluationClient } from './evaluationClient';
+
+export class LimitDisclosureEvaluationHandler extends AbstractEvaluationHandler {
+  constructor(client: EvaluationClient) {
+    super(client);
+  }
+
+  public getName(): string {
+    return 'LimitDisclosureEvaluation';
+  }
+
+  //TODO: what is the necessary field? "@context", "credentialSchema", "credentialSubject", "type"
```

Comment:
Is the comment still necessary? Can we remove it?

Reply:
I still don't know if this is the correct assumption, maybe @hrehman-sphereon can help here